docs/indicator-configuration.md

```diff
@@ -0,0 +1,470 @@
+
+## Note about indicator configuration file formats
+
+By default, Open SDG platforms expect the individual indicator configuration files to be uploaded in YML format to the "indicator-config" folder. See this [example in the data starter repo](https://github.com/open-sdg/open-sdg-data-starter/blob/develop/indicator-config/1-1-1.yml)
+
+### Indicator configuration forms
+To help with configuring indicators, you can optionally enable user-friendly indicator configuration forms, which will allow you to fill in a form and then download a YML file which will be ready to upload to your platform.
+
+For more details on how to enable these forms, see the [indicator_config_form site configuration setting](configuration.md#indicator_config_form).
+
+## Note about translation keys
+Indicator configuration values can either be filled in with normal text ("My field value") or with [translation keys](glossary.md#translation-keys) (my_translations.my_translation). In the examples below, we will try to demonstrate both possibilities.
+
+As an optional shorthand, if the translation key is in the `data` group, then the group can be omitted. For example, the translation key `data.female` can be written as simply `female`.
+
+## Note about unit-specific and series-specific settings
+
+Several indicator configuration settings can be limited to a particular Unit and/or Series. For example, the `graph_titles` setting can be configured like this:
+
+```
+graph_titles:
+  - unit: Percent
+    title: My title for percent
+  - unit: Total
+    title: My title for total
+```
+
+Indicator setting for a specific Series-Unit combination can be configured like this:
+
+```
+graph_titles:
+  - series: Wastewater treated
+    unit: Percent
+    title: My title for percent of wastewater treated
+```
+
+In addition to graph_titles, the other fields like this include:
+
+* graph_annotations
+* graph_limits
+* precision
+
+These fields are described below. Note that if a unit/series is not specified, then the item will apply to any unit/series. For example:
+
+```
+graph_titles:
+  - series: SERIES123
+    title: This title will appear for SERIES123 only
+  - series: ''
+    title: This title will appear for all other series
+```
+
+## Indicator configuration fields
+
+### composite_breakdown_label
+
+**_Optional_** This setting is used when importing data from SDMX to specify a more useful label for the COMPOSITE_BREAKDOWN column (if present). Translation keys are supported, as always. Using the example below would change the COMPOSITE_BREAKDOWN label to "Hazard type" for this indicator:
+
+```nohighlight
+composite_breakdown_label: Hazard type
+```
+
+### computation_units
+
+**_Recommended_**: This setting is used to specify the unit used in the headline data for the indicator.
```

Comment:
Do you think we should explicitly say where it will appear? In this case, the value will appear in the chart/table footers (just like Copyright below).